Resources/datasets/data-point-attributes/fundamentals/enums.html

```diff
@@ -7,8 +7,8 @@ <h4>FineFundamental Attributes</h4>
 <h4>MorningstarSectorCode Enumeration</h4>
 <p>Sectors are large super categories of data. To access the sector of an Equity, use the <b>MorningstarSectorCode</b> property.</p>
 <div class="section-example-container">
-    <pre><code class="language-cs">filteredFine = fine.Where(x =&gt; x.AssetClassification.MorningstarSectorCode == MorningstarSectorCode.Technology);</code></pre>
-    <pre><code class="language-python">filtered_fine = [x for x in fine if x.AssetClassification.MorningstarSectorCode == MorningstarSectorCode.Technology]</code></pre>
+    <pre class="csharp">filteredFine = fine.Where(x =&gt; x.AssetClassification.MorningstarSectorCode == MorningstarSectorCode.Technology);<pre class="python"></pre>
+    <pre class="python">filtered_fine = [x for x in fine if x.AssetClassification.MorningstarSectorCode == MorningstarSectorCode.Technology]<pre class="python"></pre>
```

Comment:
The opening tags (`<pre class="_____">`) look good. We just need to fix the closing tag (at the end of the line) to be `</pre>` instead of `<pre class="python">`.